src/main/scala/com/redis/sentinel/SentinelCluster.scala

```diff
@@ -137,6 +137,17 @@ class SentinelCluster (clusterConfig: SentinelClusterConfig = SentinelClusterCon
     }
   }
 
+  def getHeartBeatStatus: Boolean = {
+    if (clusterConfig.heartBeatEnabled) {
+      this.synchronized {
+        // At least one sentinel needs to be alive.
+        sentinelMonitors.values.map{_.isHeartBeating}.exists(_ == true)
```

Comment:
Ah yeah I can contract that.. But I can't use forall cause that will return true on an empty list.